content/pt/docs/concepts/services-networking/network-policies.md

```diff
@@ -0,0 +1,248 @@
+---
+title: Políticas de redes
+content_type: concept
+weight: 50
+---
+
+<!-- overview -->
+
+Se você deseja controlar o fluxo de tráfego no nível de endereçamento IP ou
+porta (camadas OSIs 3 e 4), então você deve considerar usar o objeto _NetworkPolicy_
+do Kubernetes para algumas aplicações em seu cluster.
+Os objetos de políticas de redes (_NetworkPolicies_) são orientadas à experiência do 
```

Comment:
Maybe we can have a different way to express this, I was thinking of "As políticas de rede (_NetworkPolicies_) são funcionalidades centradas nas aplicações. Estas, permitem que o usuário (desenvolvedor) especifique as permissões que um {{< glossary_tooltip text="pod" term_id="pod">}} tem para se comunicar com diversas outras entidades de rede."